backend/main/models/community_users.go

```diff
@@ -42,12 +64,12 @@ func GetUsersForCommunity(db *s.Database, communityId, start, count int) ([]Comm
 	if err != nil && err.Error() != pgx.ErrNoRows.Error() {
 		return nil, 0, err
 	} else if err != nil && err.Error() == pgx.ErrNoRows.Error() {
-		return []CommunityUser{}, 0, nil
+		return []CommunityUserType{}, 0, nil
 	}
 
 	// Get total number of users
 	var totalRecords int
-	countSql := `SELECT COUNT(*) FROM community_users WHERE community_id = $1`
+	countSql := `SELECT COUNT(*) FROM (SELECT addr FROM community_users WHERE community_id = $1 group BY community_users.addr) as temp_users_addr`
 	_ = db.Conn.QueryRow(db.Context, countSql, communityId).Scan(&totalRecords)
 
 	return users, totalRecords, nil
```

Comment:
This function is used here: https://github.com/DapperCollectives/CAST/blob/958b9eea7f77c01aa3c3378da7df373d0d843da6/backend/main/server/app.go#L1498-L1499
the endpoint allows filtering, that with change will have two types of return payload. They question is if this should be implemented on a separate endpoint.

Reply:
i think it probably should be a separate endpoint, but we can merge this in as-is and i'll create a backlog ticket to refactor.